pkg/controller/istio/istio_controller.go

```diff
@@ -238,7 +238,7 @@ func (r *ReconcileConfig) reconcile(logger logr.Logger, config *istiov1beta1.Ist
 	if config.Status.Status == istiov1beta1.Reconciling {
 		logger.Info("cannot trigger reconcile while already reconciling")
 		return reconcile.Result{
-			Requeue:      true,
+			Requeue:      false,
```

Comment:
If this is explicitly set to `true` then requeuing happens immediately instead of after the given timeframe? Because this is what I understand from your PR description but this is not what I'd understand from the description [here](https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/reconcile/reconcile.go#L31). Also, if I check the [code](https://github.com/kubernetes-sigs/controller-runtime/blob/b9f30df3f3a81bad9902b8bec5d92622a363bd4f/pkg/internal/controller/controller.go#L216), I would think that the `Requeue` field's value should not have any effect if `RequeueAfter` is set to a non zero value.

Reply:
It seems that you're right :)

My initial assumption about how this works was right after all, I close this PR.